tests/style_test.rs

```diff
@@ -0,0 +1,54 @@
+extern crate yoga;
+
+use yoga::{Direction, FlexDirection, Node, StyleUnit, Undefined};
+
+#[test]
+fn test_copy_style_same() {
+	let node0 = Node::new();
+	let node1 = Node::new();
+	assert!(!node0.is_dirty());
+
+	node0.copy_style(&node1);
+	assert!(!node0.is_dirty());
+}
+
+#[test]
+fn test_copy_style_modified() {
+	let node0 = Node::new();
+	assert!(!node0.is_dirty());
+	assert_eq!(FlexDirection::Column, node0.get_flex_direction());
+
+	match node0.get_style_max_height() {
+		StyleUnit::UndefinedValue => assert!(true),
+		_ => assert!(false),
+	}
+
+	let mut node1 = Node::new();
+	node1.set_flex_direction(FlexDirection::Row);
```

Comment:
I know currently the imports are kind of annoying to get the `style!` macro working, but the other tests use it so it would be nice to use here as well.

Later I need to add something like a `yoga::prelude` module to wrap up most of the things you would typically `use` from yoga.

Reply:
To use the `style!` macro, you need to `#[macro_use]` yoga, then use `yoga::Point` and `yoga::FlexStyle::*`

I know, it's not the best right now :P

Reply:
Do you feel strongly about this? It's much easier to port things over without using the macro, and I feel like the macro itself is very well tested already. Or we can introduce a special test for the macro to verify all branches if that's necessary.

Reply:
I don't feel too strongly about it, and at first I didn't realize you were porting things over the way you are. So I think for now it's totally fine, I may go through later and change them to use the macro as it illustrates a more typical way to use the library.